src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -198,55 +198,56 @@ public String getJsonString() {
 
     @Override
     public List<String> getInvalidityInfo() {
-        FieldValidator validator = new FieldValidator();
         List<String> errors = new ArrayList<>();
 
         // Check for null fields.
 
-        addNonEmptyError(validator.getValidityInfoForNonNullField(
-                FieldValidator.FEEDBACK_SESSION_NAME_FIELD_NAME, feedbackSessionName), errors);
+        addNonEmptyError(FieldValidator.getValidityInfoForNonNullField(
+                    FieldValidator.FEEDBACK_SESSION_NAME_FIELD_NAME, feedbackSessionName), errors);
 
-        addNonEmptyError(validator.getValidityInfoForNonNullField(FieldValidator.COURSE_ID_FIELD_NAME, courseId), errors);
+        addNonEmptyError(FieldValidator.getValidityInfoForNonNullField(
```

Comment:
It might be better to just move `FieldValidator.getValidityInfoForNonNullField(` to the next line as well